app/src/main/java/io/intrepid/contest/screens/contestcreation/reviewcontest/ReviewContestFragment.java

```diff
@@ -0,0 +1,77 @@
+package io.intrepid.contest.screens.contestcreation.reviewcontest;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.widget.TextView;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BaseFragment;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.screens.contestcreation.ContestCreationFragment;
+import io.intrepid.contest.screens.contestcreation.EditContestContract;
+import io.intrepid.contest.screens.contestcreation.categorieslist.CategoryAdapter;
+
+public class ReviewContestFragment extends BaseFragment<ReviewContestPresenter> implements ReviewContestContract.View, ContestCreationFragment {
+    private static final String CONTEST_KEY = "CONTEST_KEY";
+    @BindView(R.id.contest_title_button)
+    TextView titleButton;
+    @BindView(R.id.contest_description_button)
+    TextView descriptionButton;
+    @BindView(R.id.categories_recycler_view)
+    RecyclerView categoriesRecyclerView;
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+    }
+
+    @Override
+    protected int getLayoutResourceId() {
+        return R.layout.fragment_review_contest;
+    }
+
+    @NonNull
+    @Override
+    public ReviewContestPresenter createPresenter(PresenterConfiguration configuration) {
+        Contest.Builder contestBuilder = ((EditContestContract) getActivity()).getContestBuilder();
+        return new ReviewContestPresenter(this, configuration, contestBuilder);
+    }
+
+    @Override
+    public void showReviewPage(Contest.Builder contestBuilder) {
+        titleButton.setText(contestBuilder.title);
+        descriptionButton.setText(contestBuilder.description);
+        categoriesRecyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
```

Comment:
The recyclerview setup should be done in the `onViewCreated` method of the activity so that if another method requires a call to `view.showReviewPage()`, then the recyclerview setup isn't repeated again.